www/include/core/login/processLogin.php

```diff
@@ -97,30 +97,31 @@
     );
     if ($centreonAuth->passwdOk == 1) {
         $centreon = new Centreon($centreonAuth->userInfos);
+        // security fix - regenerate the sid after the login to prevent session fixation
+        session_regenerate_id();
```

Comment:
```suggestion
        session_regenerate_id(true);
```

Reply:
Session should be destroyed as we do not delete nor handle the destroyed session.